docs/docs/guides/airy-core-in-production.md

```diff
@@ -0,0 +1,139 @@
+# Running the Airy Core Plaform in production
+
+
+The cloud providers part can be left as TODO in the first pr. It's enough to cover all the requirements (size of a kubernetes cluster, how to get the code up and running in the cluster, how to run the services (redis, postgres, kafka, zookeeper) both in the same kubernets cluster or how to configure the apps so that the system works)
+
+## Architecture
+
+The Airy Core Platform is messaging platform, based on microservices and comprised of a backend and frontend system. 
+
+The `backend` system is a streaming platform and its role is to:
+- Receive messages or events from different sources, process them on the fly and store them in Kafka `(sources)`
+- Make them available and accessible through an API (api)
+- Send messages or events to external parties (webhooks)
+
+Apart of the messaging infrastructure, there is also an `admin` app which manages users, authentication and the other features related to the organization using the Airy Core Platform and its users.
+
+The `frontend` system is comprised of a webapp and a custom Airy source called the `Chat Plugin`.
+
+Having that in mind, the following docker containers or `Airy apps` need to run in order for the Airy Core Platform to be fully functional:
+
+Sources:
+- sources-facebook-webhook - Receive messages from Facebook
+- sources-facebook-events-router - Process messages from Facebook
+- sources-facebook-sender - Send messages to the Facebook source
+- sources-google-webhook - Receive messages from Google
+- sources-google-events-router - Process messages from Google
+- sources-google-sender - Send messages to the Google source
+- sources-chatplugin - Receive and processing messages from the Airy Chat Plugin source
```

Comment:
I think instead of enumerating the actual apps, I would present the convention we use (the doc will inevitably run out of data as we add sources otherwise)

Reply:
I thought that the name in ever bullet should link to a documentation for the particular app (docker registry, variables, ports, liveness-probe). This can also be a part of a separate `architecture` document.

For the documentation of the individual apps, Github packes isn't good enough, if we just link to the repository. So perhaps we use a separate document for every app or hub.docker.com.

Reply:
we discussed this offline and I think we're on the same page (about conventions being more important than the specific pod)